### ISSUE TITLE
Proposal: UWP services enable their Capabilities themselves

### ISSUE THREAD

**Issue by Alexees** [Microsoft/MixedRealityToolkit-Unity#3695]
## Overview
Building the MRTK just like that throws 2 errors on UWP by default. Microphone and Spatial Perception are not part of the default PlayerSettings configuration, one has to manually edit this file.
It would be much nicer if available services would toggle these settings themselves, using the API
```
UnityEditor.PlayerSettings.WSA.GetCapability()
UnityEditor.PlayerSettings.WSA.SetCapability()
```

## Expected Behavior
There is no need to manually adjust needed capabilities if services depend on them anyways

## Actual Behavior
You run and check for errors prior to building

## Steps to reproduce
Build the InteractablesGallery scene for UWP desktop in development mode and see errors about missing capabilities thrown.

## Unity Editor Version
2018.3.9f1

## Mixed Reality Toolkit Release Version
mrtk_development

**Comment by Jarodshow** [comment]
A warning that your scene uses capabilities that aren't enabled would be SUPER useful

**Comment by davidkline-ms** [comment]
Nice suggestion, @Alexees! We'll add it to our backlog.

**Comment by keveleigh** [comment]
Should be resolved via #3742. Please reopen if it doesn't work as expected.
